FRAuth/FRAuth/DeviceBinding/UserKeySelector.swift

```diff
@@ -56,3 +56,25 @@ public class DefaultUserKeySelector: NSObject, UserKeySelector {
 
 /// Completion Callback for selected user key
 public typealias UserKeySelectorCallback = (_ selectedUserKey: UserKey?) -> Void
+
+/// Format the createdAt before display in actionsheet
+extension Double {
+    func format(dateFormat: String = "yyyyMMdd HH:mm:ss") -> String {
```

Comment:
I would suggest a more specific name for this method. _format_ is too generic and also if it returns a value instead of modifying the underlying entity, we would call it _formatted_ . So I suggest something like **formattedDateString**

Reply:
addressed this one